web/craco.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  eslint: {
+    enable: false
+  },
```

Comment:
by default creat react app runs eslint itself on each build. but since our ci is always doing that it seems unnecessary. this will speed up the build. plus it uses different eslint version so comes up with different problems.

Reply:
Nice catch, is it possible to add that explanation as a comment in the file?